docs/tokens.md

```diff
@@ -3,137 +3,239 @@ id: tokens
 
-You've probably heard of the **ERC20** standard, and that's why you're here.
+You've probably heard of the [ERC20](#erc20) or [ERC721](#erc721) token standards, and that's why you're here.
+
+---
 
 ## ERC20
 
-An ERC20 token is a contract that keeps track of a `mapping(address => uint256)` that represents a user's balance. These tokens are _fungible_ in that any one token is exactly equal to any other token; no tokens have special rights or behavior associated with them. This makes ERC20 useful for things like a medium of exchange currency, general voting rights, staking, and more.
+An ERC20 token contract keeps track of [_fungible_ tokens](#different-kinds-of-tokens): any one token is exactly equal to any other token; no tokens have special rights or behavior associated with them. This makes ERC20 tokens useful for things like a **medium of exchange currency**, **voting rights**, **staking**, and more.
+
+OpenZeppelin provides many ERC20-related contracts. On the [`API reference`](api/token/ERC20) you'll find detailed information on their properties and usage.
+
+### Constructing an ERC20 Token Contract
+
+Using OpenZeppelin, we can easily create our own ERC20 token contract, which will be used to track _Gold_ (GLD), an internal currency in a hypothetical game.
+
+Here's what our GLD token might look like.
+
+```solidity
+pragma solidity ^0.5.0;
+
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/ERC20Detailed.sol";
+
+contract GLDToken is ERC20, ERC20Detailed {
+    constructor(uint256 initialSupply) ERC20Detailed("Gold", "GLD", 18) public {
+        _mint(msg.sender, initialSupply);
+    }
+}
+```
+
+OpenZeppelin contracts are often used via [inheritance](https://solidity.readthedocs.io/en/latest/contracts.html#inheritance), and here we're reusing the [`ERC20`](api/token/ERC20#erc20) (for the basic standard implementation) and [`ERC20Detailed`](api/token/ERC20#erc20detailed) (to get the [`name`](api/token/ERC20#ERC20Detailed.name()), [`symbol`](api/token/ERC20#ERC20Detailed.symbol()) and [`decimals`](api/token/ERC20#ERC20Detailed.decimals()) properties) contracts. Additionally, we're creating an `initialSupply` of tokens, which will be assigned to the address that deploys the contract.
+
+_For a more complete discussion of ERC20 supply mechanisms, see [our advanced guide](https://forum.zeppelin.solutions/t/how-to-implement-erc20-supply-mechanisms/226/)_.
+
+That's it! Once deployed, we will be able to query the deployer's balance:
+
+```javascript
+> GLDToken.balanceOf(deployerAddress)
+1000
+```
+
+We can also [transfer](api/token/ERC20#IERC20.transfer(address,uint256)) these tokens to other accounts:
 
-OpenZeppelin provides a few different ERC20-related contracts. Here are the core contracts you'll almost definitely be using:
+```javascript
+> GLDToken.transfer(otherAccount, 300)
```

Comment:
Why <code>deployer<b>Address</b></code> but <code>other<b>Account</b></code>? :stuck_out_tongue:

Reply:
Haha guilty as charged. Should we use address or account? For a beginner guide, and given that `address` is a thing in Solidity, I think keeping it simple is best.

Reply:
"keeping it simple" = "address"? 

That argument sounds convincing to me.